python/tk_multi_about/context_browser.py

```diff
@@ -80,15 +80,14 @@ def process_result(self, result):
             
             i = self.add_item(browser_widget.ListItem)
             details = []
-            details.append("<b>Project %s</b>" % d.get("name"))
-
-            # add the site url
-            link_color = self._app.engine.style_constants["SG_HIGHLIGHT_COLOR"]
-            full_site_url = result.get("shotgun_url") if result.get("shotgun_url") else ""
-            nice_name_site_url = self._get_url_nice_name(full_site_url)
-            site_display_template = "<a href=\"{url}\" style=\"color:{color}\" >{display_url}</a>"
-            site_str = site_display_template.format(url=full_site_url, display_url=nice_name_site_url, color=link_color)
-            details.append(site_str)
+
+            # get the site url
+            full_site_url = result.get("shotgun_url", "")
```

Comment:
When is `result["shotgun_url"]` None? Don't we always have a site? Do we really need the else clause? If we do, please add a comment to explain that case - it's not totally obvious why it's needed!

If it is needed, please do `result.get("shotgun_url")` so that it either returns None or the url. No need to have an empty string case i think.